tests/libvmi/vmi.go

```diff
@@ -38,7 +38,10 @@ type Option func(vmi *v1.VirtualMachineInstance)
 func New(opts ...Option) *v1.VirtualMachineInstance {
 	vmi := baseVmi(randName())
 
-	WithTerminationGracePeriod(0)(vmi)
+	// The behavior for pods with spec.terminationGracePeriod: 0
```

Comment:
Let's also document `New` and factory functions `NewCirros`,...

Reply:
done